app/src/main/java/io/intrepid/contest/screens/contestcreation/describecontest/DescribeContestFragment.java

```diff
@@ -9,6 +9,7 @@
 import io.intrepid.contest.R;
 import io.intrepid.contest.base.BaseFragment;
 import io.intrepid.contest.base.PresenterConfiguration;
+import io.intrepid.contest.customviews.HintLabelEditText;
```

Comment:
In the meantime, I think it can remain as HintLabelEditText. We don't really care that it's a custom view. We just care that it contains a text field with some means of entering text and getting text back. Composition vs. inheritance doesn't make much of a difference in this case. If we need to change it in the future, we should do that. The changes were just so we are using the same styles and custom views.